### ISSUE TITLE
Project is dead?

### ISSUE THREAD

**Issue by ghost** [perseo22/pacmanager#57]
Hi,

the project is still alive?

**Comment by clickwir** [comment]
Hasn't been any decent update in 6+ months. Seems like a great project, but needs to continue to be maintained. There don't appear to be many alternatives to this style connection manager.

**Comment by ghost** [comment]
yep.. totally agree.. we need a new developer to fork and keep update the project..

**Comment by perseo22** [comment]
Yeah, or simply make fucking donations to make the only developer see his work matters....




On Mon, Jul 10, 2017 at 7:12 PM +0200, "Karl Fiabeschi" <notifications@github.com> wrote:












yep.. totally agree.. we need a new developer to fork and keep update the project..

**Comment by Prototype-X** [comment]
Add donate button https://gist.github.com/skratchdot/6668394177f5444bab26

**Comment by perseo22** [comment]
No.




On Mon, Jul 10, 2017 at 8:07 PM +0200, "Maxim" <notifications@github.com> wrote:












Add donate button https://gist.github.com/skratchdot/6668394177f5444bab26

**Comment by Prototype-X** [comment]
Why? It will be easier to make a donation.

**Comment by perseo22** [comment]
I fucking hate donation buttons. I donate when I think I have to, whether there is a button or not. I am very tired. Chat finished.




On Mon, Jul 10, 2017 at 8:23 PM +0200, "Maxim" <notifications@github.com> wrote:












Why? It will be easier to make a donation.

**Comment by ghost** [comment]
@perseo22 If you don't talk nobody can talk for you

The problem is donation? the problem is free time? The problem is perl?  etc etc

Please let the community know the issues so the community can help you for developing PAC.

**Comment by gfrenoy** [comment]
If David is tired, it's probably time to find new collaborators for the project.  There are so many ways to make the project attractive for donors ; giving negative answers is one of the worst :/

If communication is broken and, for whatever reason that we will respect, a collaboration is not possible, well, ... it will be time for a fork and a new core-team.

**Comment by dandv** [comment]
> I fucking hate donation buttons. I donate when I think I have to, whether there is a button or not.

David, what happened that day? If you think you have to donate, how do you donate without a button?

If other users want to donate, how will they know donations are possible? I would like to donate $100 (more than SecureCRT costs) because I believe in open source.

I also believe in being nice to people, and [accepting simple pull requests](https://github.com/perseo22/pacmanager/pull/50) :)

**Comment by Prototype-X** [comment]
> I also believe in being nice to people, and accepting simple pull requests :)

In fact, the project is no longer supported and is not being developed by the author. So it's better to look for a new developer or take the development on yourself.

**Comment by xenithorb** [comment]
@perseo22 Are you willing to continue development AND/OR include members of this community into the project as committers? 

As a Fedora maintainer I have a responsibility to work with upstream, but also one to the users to help keep this software thriving. 

You have users here @KarlFiabeschi @Prototype-X @dandv @gfrenoy that seem to want to keep your hard work alive and I'm inclined to help them find an outlet for them to do so.

I don't really have time for games, and neither do you apparently. Do you want people and yourself developing on this anymore as `pacmanager` or should we take our stuff elsewhere?

**Comment by gfrenoy** [comment]
As we did not hear anything from David, we made a fork of this project and rebranded it to [Ásbrú Connection Manager](https://asbru-cm.net/).

We have reached a point where the software is working and is correctly packaged for Debian/Ubuntu.  A release candidate is available for testing.

We are now looking for help and support to finalize tests, package it for other distributions and think about the future of this great piece of software :)

If you can help or just want to try an alternative, please join us on https://github.com/asbru-cm/asbru-cm !

**Comment by karlbaillie** [comment]
So you want us to "fucking donate" yet don't provide any clear method to donate...

Some quality community interaction right there.

**Comment by gentunian** [comment]
@kjbweb I've donated 4 or 5 times already (modest donations 10-15 dollars) and I didn't need a donate button in github. You could donate via its website, there you have a donate button for you:

https://sites.google.com/site/davidtv/

Maybe David is tired of receiving nothing from community. This tool is a pro tool that I've been used since the last 4 years. Project may not be updated but for the last 4 years of using it I've only have 2 issues that has been already solved.

Contribute to the project, don't let his work die in vain, for the sake of open source give him a chance :)

**Comment by clickwir** [comment]
I've switched over to Asbru CM. https://github.com/asbru-cm/asbru-cm

**Comment by glove400** [comment]
"I fucking hate donation buttons. I donate when I think I have to, whether there is a button or not. I am very tired. Chat finished."

Are you a fucking toddler? lol

**Comment by jav-12** [comment]
It is a shame to hear that from @perseo22 I remember I contacted him via Teamviewer to get some help from him with PAC manager. At least the community is big to maintain this wonderful job alive. It's like 7 years using it without any issues. It is time to use the new version that was forked. Thanks anyway.
